source/fundamentals/collations.txt

```diff
@@ -4,5 +4,298 @@ Collations
 
 .. default-domain:: mongodb
 
-A collation is a set of character ordering and matching rules that
-apply to a specific language and locale.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **collations** to run
+operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
```

Comment:
I edited this part to introduce the term first, then define it in the next sentence. I used "character ordering conventions" in the definition instead of just "string ordering."